test/metabase/api/collection_test.clj

```diff
@@ -103,10 +103,42 @@
                 "Rasta Toucan's Personal Collection"
                 "Trash Bird's Personal Collection"]
                (->> (mt/user-http-request :crowberto :get 200 "collection")
+                    (filter #((set (map mt/user->id [:crowberto :lucky :rasta :trashbird])) (:personal_owner_id %)))
+                    (map :name)
+                    sort))))
+
+      (testing "...or we are *admins* but exclude other user's collections"
+        (is (= ["Crowberto Corv's Personal Collection"]
+               (->> (mt/user-http-request :crowberto :get 200 "collection" :exclude-other-user-collections true)
                     (filter #((set (map mt/user->id [:crowberto :lucky :rasta :trashbird])) (:personal_owner_id %)))
                     (map :name)
                     sort)))))
 
+    (testing "You should only see your collection and public collections"
+      (let [admin-user-id  (u/the-id (test.users/fetch-user :crowberto))
+            crowberto-root (db/select-one Collection :personal_owner_id admin-user-id)]
+        (mt/with-temp* [Collection [collection]
```

Comment:
nitty: we're moving to toucan 2 and it's recommended to use `t2.with-temp/with-temp` now. we should do this with new tests because it takes quite an effort to rewrite between the 2 versions

Reply:
Done